From ddfcc907907a20d9f8fbc1416492e2093b339b22 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 26 Jun 2023 08:46:26 +0000
Subject: [PATCH] IWYU: add cstddef for size_t in bookmarks::UrlLoadStats

Bug: 957519
Change-Id: Ic58b8b6adc001f0931c2d1d74b7ac1ac3014fbc1
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4644168
Auto-Submit: Stephan Hartmann <stha09@googlemail.com>
Reviewed-by: Mikel Astiz <mastiz@chromium.org>
Commit-Queue: Mikel Astiz <mastiz@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1162286}
---

diff --git a/components/bookmarks/common/url_load_stats.h b/components/bookmarks/common/url_load_stats.h
index b4d975b..b5ddece 100644
--- a/components/bookmarks/common/url_load_stats.h
+++ b/components/bookmarks/common/url_load_stats.h
@@ -5,6 +5,7 @@
 #ifndef COMPONENTS_BOOKMARKS_COMMON_URL_LOAD_STATS_H_
 #define COMPONENTS_BOOKMARKS_COMMON_URL_LOAD_STATS_H_
 
+#include <cstddef>
 #include <cstdint>
 #include <vector>
 
@@ -51,4 +52,4 @@
 
 }  // namespace bookmarks
 
-#endif  // COMPONENTS_BOOKMARKS_COMMON_URL_LOAD_STATS_H_
\ No newline at end of file
+#endif  // COMPONENTS_BOOKMARKS_COMMON_URL_LOAD_STATS_H_
